zap/src/main/java/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -733,7 +733,7 @@ private ZapTextField getTxtProxyChainUserName() {
     /**
      * This method initializes txtProxyChainPassword
      *
-     * @return org.zaproxy.zap.utils.ZapTextField
+     * @return JPasswordField that will be used to gather the password from the user
```

Comment:
The type can be removed (e.g. `The field that will ...`), that's already provided in the method signature.